src/appleseed.python/bindmaterial.cpp

```diff
@@ -47,10 +47,22 @@ using namespace std;
 namespace
 {
     auto_release_ptr<Material> create_material(
-        const string&   name,
-        const bpy::dict& params)
+        const std::string& material_type,
```

Comment:
Remove redundant qualifier, and rename `material_type` to `model` since that's the terminology used everywhere in appleseed.